### PR TITLE
Add i18n support

### PR DESCRIPTION
**Pull request by BeautyYuYanli**
I created an i18n framework. Chinese (simplified & traditional) are supported now.

relate issue: #89

image
image

### CODE REVIEW

**Comment by MrGlockenspiel** [comment]
very nice

**Comment by ReperakDev** [comment]
This leaks memory

**Comment by BeautyYuYanli** [comment]
> This leaks memory

Can you explain how this happens? Should I write `i18n_map` outside the source file?

File: src/i18n.c [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+char *i18n_map[i18n_map_size][4] = {
+    {"zh_CN", "激活 ", "转到“设置”以激活 ", "。"},
+    {"zh_TW", "啟用 ", "移至[設定]以啟用 ", "。"},
+    {"zh_HK", "啟用 ", "移至[設定]以啟用 ", "。"},
+};
+char *another_strcat(size_t len, char *strings[]){
+    unsigned long t = 0;
+    for (size_t i = 0; i < len; ++i)
+        t += strlen(strings[i]);
+    char *tmp = (char*)malloc((t+1) * sizeof(char));
```
Comment (ReperakDev) [inline src/i18n.c:15]: This malloc is never freed

Comment (BeautyYuYanli) [reply src/i18n.c:15]: But the string needs to be passed to the drawer. It should behave like a literal string stored in the heap.

Comment (ReperakDev) [reply src/i18n.c:15]: `i18n` in activate_linux.c is not freed at the end of the program.

Comment (BeautyYuYanli) [reply src/i18n.c:15]: So what's the better practice? Can I declare `i18n_info.title` as a char array with a finite length? Or is there any way to create something behaving the same as literal string?

Comment (ReperakDev) [reply src/i18n.c:15]: Just freeing it at the end of the program is good enough, I believe.

**Pull request merged**